english/turing-test/turing-test.md

```diff
@@ -0,0 +1,13 @@
+After realizing that the marshmallow seems to be self-aware, you have decided to confirm your suspision by delivering the marsmallow a turing test. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.